utils/zookeeper-dump-tree/main.cpp

```diff
@@ -79,7 +80,11 @@ int main(int argc, char ** argv)
                     throw;
             }
 
-            std::cout << it->first << '\t' << response.stat.numChildren << '\t' << response.stat.dataLength << '\n';
+            std::cout << it->first << '\t' << response.stat.numChildren << '\t' << response.stat.dataLength;
+            if (options.count("ctime")) {
```

Comment:
Let's make a bool variable and move it out of the loop.
Most likely there will be no difference but lookup in a map by string in a loop just doesn't look good.

Reply:
Okay, let's try.